tool/pom.xml

```diff
@@ -216,11 +216,6 @@
     </build>
 
     <dependencies>
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-lang3</artifactId>
-            <scope>provided</scope>
```

Comment:
@bvfalcon To avoid 'NoClassDefFoundError ArrayUtils' exception, you can just remove <scope>provided</scope> here. We are OK with this dependency being added to the elytron-tool jar.